apps/products-feed/src/pages/api/feed/[url]/[channel]/google.xml.ts

```diff
@@ -226,13 +230,11 @@ export const handler = async (req: NextApiRequest, res: NextApiResponse) => {
         channel,
       });
 
-      logger.debug("Feed uploaded to S3, redirecting the download URL", {
-        downloadUrl,
```

Comment:
I think its good to log it. Its public URL (for google, scrappable, in general its products catalogue). If we have it in datadog logs, we will be able to debug app, otherwise we will ask about it